scripts/fast-reboot

```diff
@@ -522,6 +522,7 @@ then
     exit "${EXIT_FAILURE}"
 fi
 
+debug "Starting $REBOOT_TYPE"
```

Comment:
What's the log severity `debug` is using?

Reply:
notice